cincinnati/src/plugins/internal/edge_add_remove.rs

```diff
@@ -25,6 +27,10 @@ pub struct EdgeAddRemovePlugin {
 #[async_trait]
 impl InternalPlugin for EdgeAddRemovePlugin {
     async fn run_internal(self: &Self, io: InternalIO) -> Fallible<InternalIO> {
+        get_tracer()
+            .get_active_span()
+            .update_name("edge_add_remove".to_string());
```

Comment:
Use the predefined constant instead:
```suggestion
            .update_name(Self::PLUGIN_NAME.to_string());
```